andela/ah-kgl-avengers-frontend
# #165020179 | #165020180 Bookmark an article and list all my bookmarks

### What does this PR do?
User should be able to bookmark an article and see list of his/her bookmarks
### Description of Task to be completed?
- Authenticated user will be able to  bookmark an article
- Authenticated user will be able to see the list of his bookmarks 
### How should this be manually tested?
- Clone the repository locally with `git clone https://github.com/andela/ah-kgl-avengers-frontend.git` command.
- Check out to this branch by running `git checkout ft-bookmark-article-165020179`.
- Install dependencies by running `npm install`.
- Update the .env file according to the .env.sample file
- Start the server with `npm start` and put this url `http://localhost:8000/` in the address bar.
- Bookmark an article by clicking on the bookmark icon shown in the article. This article will be added to your bookmark list
- Check your bookmark by clicking on an avatar or your image profile shown on the top-right corner, from the dropdown menus, my bookmarks then the new page with your bookmark list will be displayed.
### Any background context you want to provide?
- User should have an account on `AuthorsHaven` to be able to bookmark an article.
### What are the relevant pivotal tracker stories?
[#165020179](https://www.pivotaltracker.com/story/show/165020178)
[#165020180](https://www.pivotaltracker.com/story/show/165020180)
### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/29593858/60493635-853aa880-9cad-11e9-9a27-5a774a1be565.png)

### Questions:
N/A
